sync/syncFromDisk.js

```diff
@@ -93,6 +89,15 @@ async function syncDiskToSqlWithLock(courseDir, courseId, logger) {
   } else {
     logger.info(chalk.green('✓ Course sync successful'));
   }
+
+  // Note that we deliberately log warnings/errors after syncing to the database
+  // since in some cases we actually discover new warnings/errors during the
+  // sync process. For instance, we don't actually validate exam UUIDs until
+  // the database sync step.
+  courseDB.writeErrorsAndWarningsForCourseData(courseId, courseData, (line) =>
+    logger.info(line || '')
+  );
```

Comment:
Just to be extra pedantic, but did you test it to verify that `examUUID` warnings are in fact now shown correctly on sync? (The code all looks fine to me and I agree that it _should_ display `examUUID` warnings, so I'm happy to merge as is.)

Reply:
😂 thanks for asking, I hadn't specifically tested that. After setting `checkAccessRulesExamUuid` to `true` in my local config, I do indeed see UUID warnings being printed:

<img width="1014" alt="Screenshot 2022-11-03 at 12 25 09" src="https://user-images.githubusercontent.com/1476544/199815732-6c454549-e08f-4ec6-b7f5-43258b335982.png">

Reply:
Nice, thanks for humoring me! 😄